code/upaya/prod-manifest.yml

```diff
@@ -0,0 +1,27 @@
+---
+env:
+  RAILS_ENV: production
+  RACK_ENV: production
+  SECRET_KEY_BASE: e1dfd52b4cf2ce7d416cd5c99b1344c7bf56a44567f5230836e4089faf6f099cefc3d761430de52f63bdf1365f6b2ac948f467ecb1f731d74bf72ed5aae82dd9
```

Comment:
Secrets should not be checked in to Git. Instead, we'll need to set these types of secret ENV vars using CloudFoundry's `set-env`, such as `cf set-env <APP_NAME> SECRET_TOKEN 'insert_token_here'`.